### PR TITLE
Generates report for summary of applications finalised

### PR DESCRIPTION
**Pull request by saruniitr**

### CODE REVIEW

File: api/reports/queries/standard.py [hunk 1]
```diff
@@ -369,3 +369,135 @@
...
+        , status
+    from licences_licence ll
+),
+goods as (
+select cc.reference_code         "licence_ref"
+        , sa.baseapplication_ptr_id "application_id"
+        , g.id                      "good_id"
+        , g.description             "good_desc"
+        , ct.reference              "case_type"
+        , concat(ct.reference, ' ', '(', sa.export_type, ')') "Application type formatted"
```
Comment (saruniitr) [inline api/reports/queries/standard.py:409]: how to select this in the final output starting at line479

File: api/reports/management/commands/reports.py [hunk 2]
```diff
@@ -10,13 +10,19 @@
...
+    GOODS_AND_RATINGS,
+    LICENCES_WITH_GOOD_AMENDMENTS,
+    SLA_CASES,
+    APPLICATIONS_FINALISED_SUMMARY,
+)
 
 REPORT_QUERY_LOOKUP = {
     "ogl_summary": OGL_SUMMARY,
-    "standard_applications_goods_and_ratings": GOODS_AND_RATINGS,
+    "standard_apx   plications_goods_and_ratings": GOODS_AND_RATINGS,
```
Comment (r4vi) [inline api/reports/management/commands/reports.py:22]: typo?

File: api/reports/queries/standard.py [hunk 3]
```diff
@@ -369,3 +369,138 @@
...
+applications.foi_reason "FOI Objection",
+applications.submitted_at "Submitted Datetime",
+applications.last_closed_at "Closed Datetime",
+licences.status "Licence status",
+(select string_agg(distinct advices.type, ',') from advices where advices.case_id = applications.case_id) "Outcome",
+(select string_agg(distinct goods.report_summary, ',') from goods where goods.application_id = applications.case_id) "Goods Ars",
+(select string_agg(distinct goods.ratings, ',') from goods where goods.application_id = applications.case_id) "Goods Rating",
+(select string_agg(cast((goods.good_quantity - goods.licence_usage) as text), ',') from goods where goods.application_id = applications.case_id) "Licence Remaining quantities",
+(select sum(goods.good_value) from goods where goods.application_id = applications.case_id) "Total Goods value",
+(select string_agg(distinct goods.good_desc, ',') from goods where goods.application_id = applications.case_id) "Description",
```
Comment (r4vi) [inline api/reports/queries/standard.py:495]: we split good description and name didn't we? we should `concat` the description and name here

File: api/reports/queries/standard.py [hunk 4]
```diff
@@ -369,3 +369,135 @@
...
+applications.intended_end_use "End use",
+applications.agreed_to_foi "FOI Objection flag",
+applications.foi_reason "FOI Objection",
+applications.submitted_at "Submitted Datetime",
+applications.last_closed_at "Closed Datetime",
+licences.status "Licence status",
+(select string_agg(distinct advices.type, ',') from advices where advices.case_id = applications.case_id) "Outcome",
+(select string_agg(distinct goods.report_summary, ',') from goods where goods.application_id = applications.case_id) "Goods Ars",
+(select string_agg(distinct goods.ratings, ',') from goods where goods.application_id = applications.case_id) "Goods Rating",
+(select string_agg(cast((goods.good_quantity - goods.licence_usage) as text), ',') from goods where goods.application_id = applications.case_id) "Licence Remaining quantities",
```
[the remark this answers is not in the document]
Comment (r4vi) [reply api/reports/queries/standard.py:493]: I think this is correct

**Review by r4vi — CHANGES_REQUESTED** [review]

File: api/reports/queries/standard.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (saruniitr) [reply api/reports/queries/standard.py:493]: :+1:

File: api/reports/management/commands/reports.py (hunk 2 above)
Comment (saruniitr) [reply api/reports/management/commands/reports.py:22]: yes, fixed.

File: api/reports/queries/standard.py (hunk 3 above)
Comment (saruniitr) [reply api/reports/queries/standard.py:495]: yeah this is not updated, adding name to get `name: description` and aggregate over all goods in that application

**Comment by r4vi** [comment]
you could do it in the goods table, where you create "goods_desc" so this
bit can remain unchanged

**Review by r4vi — APPROVED** [review]

**Pull request merged**